### PR TITLE
vmss instance update: support attach/detach disks on an individual instance

### PR DESCRIPTION
**Pull request by yugangw-msft**
//cc: @gatneil 

### General Guidelines

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [x] Each command and parameter has a meaningful description.
- [x] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

**Comment by promptws** [comment]
View a preview at https://prompt.ws/r/Azure/azure-cli/5539
This is an **experimental preview** for @microsoft users.

File: src/command_modules/azure-cli-vm/HISTORY.rst [hunk 1]
```diff
@@ -4,6 +4,7 @@ Release History
 ===============
 2.0.26
 ++++++
+* vmss instance update: support attach/detach disks on an individual instance
```
Comment (tjprescott) [inline src/command_modules/azure-cli-vm/HISTORY.rst]: This should be for the next version of the CLI, no?

File: src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py [hunk 2]
```diff
@@ -334,6 +334,16 @@
     short-summary: Manage data disks of a VMSS.
 """
 
+helps['vmss disk attach'] = """
+    type: command
+    short-summary: Attach manage data disks to a scale set or its instances.
```
Comment (tjprescott) [inline src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py]: Typo: "managed".

File: src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py [hunk 3]
```diff
@@ -334,6 +334,16 @@
...
+"""
+
+helps['vmss disk detach'] = """
+    type: command
+    short-summary: Detach manage data disks from a scale set or its instances.
```
Comment (tjprescott) [inline src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py]: Typo: "Managed"

File: src/command_modules/azure-cli-vm/HISTORY.rst (hunk 1 above)
Comment (yugangw-msft) [reply src/command_modules/azure-cli-vm/HISTORY.rst]: You are right. Should be the next one.

**Pull request merged**